cinema/serializers.py

```diff
@@ -1,6 +1,10 @@
+from django.db import transaction
 from rest_framework import serializers
+from rest_framework.validators import UniqueTogetherValidator
```

Comment:
The `UniqueTogetherValidator` is imported but not used in the file. If you intended to enforce unique constraints on certain fields, consider adding this validator to the appropriate serializer.